src/components/Card/Card.stories.mdx

```diff
@@ -0,0 +1,529 @@
+        />
+        <CardHeading level={3} id="heading1">
+          Card Heading
+        </CardHeading>
+        <CardContent>
+          Vestibulum id ligula porta felis euismod semper. Nulla vitae elit
+          libero, a pharetra augue.
+        </CardContent>
+      </Card>
+      <Card>
+        <CardImage
+          src="https://placeimg.com/450/250/animals"
+          alt="Alt text"
+          imageAspectRatio={CardImageRatios.TwoByOne}
+        />
+        <CardHeading level={3} id="heading1">
+          Card Heading
+        </CardHeading>
+        <CardContent>
+          Vestibulum id ligula porta felis euismod semper. Nulla vitae elit
+          libero, a pharetra augue.
+        </CardContent>
+      </Card>
+    </div>
+  </Story>
+</Preview>
+
+## Cards in a List
+
+<Preview>
+  <Story name="Cards in a List">
+    <div
+      style={{
+        display: "grid",
+        gap: "2rem",
+        "grid-template-columns": "repeat(1, 1fr)",
+      }}
+    >
+      <Card layout={CardLayouts.Horizontal} center>
+        <CardImage
+          src="https://placeimg.com/400/200/animals"
+          alt="Alt text"
+          imageAspectRatio={CardImageRatios.TwoByOne}
+        />
+        <CardHeading level={4} id="heading1">
+          Card Heading
+        </CardHeading>
+        <CardContent>
+          Praesent commodo cursus magna, vel scelerisque nisl consectetur et.
+          Sed posuere consectetur est at lobortis. Cras justo odio, dapibus ac
+          facilisis in, egestas eget quam.
+        </CardContent>
+      </Card>
+      <Card layout={CardLayouts.Horizontal} center>
+        <CardImage
+          src="https://placeimg.com/410/210/animals"
+          alt="Alt text"
+          imageAspectRatio={CardImageRatios.TwoByOne}
+        />
+        <CardHeading level={4} id="heading2">
+          Card Heading
+        </CardHeading>
+        <CardContent>
+          Praesent commodo cursus magna, vel scelerisque nisl consectetur et.
+          Sed posuere consectetur est at lobortis. Cras justo odio, dapibus ac
+          facilisis in, egestas eget quam.
+        </CardContent>
+      </Card>
+      <Card layout={CardLayouts.Horizontal} center>
+        <CardImage
+          src="https://placeimg.com/420/220/animals"
+          alt="Alt text"
+          imageAspectRatio={CardImageRatios.TwoByOne}
+        />
+        <CardHeading level={4} id="heading3">
+          Card Heading
+        </CardHeading>
+        <CardContent>
+          Praesent commodo cursus magna, vel scelerisque nisl consectetur et.
+          Sed posuere consectetur est at lobortis. Cras justo odio, dapibus ac
+          facilisis in, egestas eget quam.
+        </CardContent>
+      </Card>
+    </div>
+  </Story>
+</Preview>
+
+## Custom Image Component
+
+Instead of passing a path for the `src` prop, you can pass a custom image component in using the `component` prop.
+
+<Preview withToolbar>
+  <Story name="Custom Image Component">
+    {(args) => (
```

Comment:
This story has no custom args so this can be removed.

Reply:
Done.